src/videojs-http-streaming.js

```diff
@@ -546,7 +622,7 @@ const HlsSourceHandler = function(mode) {
       tech.hls = new HlsHandler(source, tech, localOptions);
```

Comment:
Do we want to make this be `tech.vhs` and point `tech.hls = tech.vhs` for backwards comp?